BunqSdk/Http/ApiClient.cs

```diff
@@ -49,15 +49,42 @@ public class ApiClient
         /// </summary>
         private const string DELIMITER_HEADER_VALUE = ",";
 
-        private static HttpClient client;
+        private readonly HttpClient client;
 
         private readonly ApiContext apiContext;
 
         public ApiClient(ApiContext apiContext)
         {
             this.apiContext = apiContext;
+            client = CreateHttpClient();
         }
 
+        private HttpClient CreateHttpClient()
+        {
+            return new HttpClient(CreateHttpClientHandler())
+            {
+                BaseAddress = new Uri(apiContext.GetBaseUri())
+            };
+        }
+
+        private HttpClientHandler CreateHttpClientHandler()
+        {
+            // TODO: Add HTTP Public Key Pinning. It is needed to prevent possible man-in-the-middle attacks using
```

Comment:
Is this todo supposed to be here ? If so could you remove it and create an issue for it with a link to this particular file and line ?